srcpkgs/libtracefs/patches/musl.patch

```diff
@@ -0,0 +1,15 @@
+--- a/include/tracefs.h
++++ b/include/tracefs.h
+@@ -10,6 +10,12 @@
+ #include <sched.h>
+ #include <event-parse.h>
+ 
++#ifndef __GLIBC__
```

Comment:
`man 3 CPU_SET` It's documented in the next line ;) All `cpu_set_t` related information was put into this man-page.

Reply:
https://man7.org/linux/man-pages/man3/CPU_COUNT.3.html#DESCRIPTION